ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/individual-journey/components/hearing-view-base.component.spec.ts

```diff
@@ -19,7 +19,7 @@ class StubVideoViewComponent {
 }
 
 const canCreateHearingViewComponent = <T>(component: Type<T>): void => {
-  const videoUrls = [ 'inHearingExampleVideo', 'judgeSelfViewVideo', 'otherParticipantExampleVideo' ];
+  const videoUrls = [ 'getVideoFileUrl'];
```

Comment:
I think it makes more sense to inline this now, I had it as an array before just because it was very long on one string but now it's not "videoUrls" anymore either, it's an actual method call

Reply:
Done